doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -91,7 +91,7 @@ See `Matter samples`_ for the list of changes for the Matter samples.
 Matter fork
 +++++++++++
 
-The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, the ``1.0.0.2`` tag.
+The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, the ``SVE RC2`` tag.
 
   * Added:
```

Comment:
If you could move this line and lines 98-101 to the Matter section above, line 87. I messed up when editing the PR that added these lines.